src/main/java/Application/OpenCVServer/ServerRunner.java

```diff
@@ -0,0 +1,35 @@
+package Application.OpenCVServer;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import java.io.IOException;
+import java.io.InvalidObjectException;
+
+public class ServerRunner implements Runnable {
+    private OpenCVServerOperator activeOperator;
+
+    public ServerRunner() {
+        activeOperator = OpenCVServerOperator.getInstance();
+    }
+    @Override
+    public void run() {
+        while(true) {
```

Comment:
Check out the comments on the ServerOperator to maybe modify this loop (dont do everything inside). Also, have in mind this loop can be exited so the thread woul never end (join) unless an exception happens or is killed

Reply:
This is a Docker application, so no need for this.
But anyways just add a TODO here